index.js

```diff
@@ -103,7 +106,8 @@ if (isUpload) {
 
 if (isPublish) {
     spinnerStart('Publishing');
-    publish({ apiConfig }).then(res => {
+
+    publish({ apiConfig }, trustedTesters ? "trustedTesters" : undefined ).then(res => {
```

Comment:
Mind changing this to just:

```js
publish({ apiConfig }, trustedTesters && "trustedTesters").then(res => {
```

`trustedTesters` should already be `undefined` if not passed in.

Happy to merge/release once you make that change. Really appreciate your contribution!